scipy/odr/models.py

```diff
@@ -108,6 +108,27 @@ def polynomial(order):
     polynomial : Model instance
         Model instance.
 
+    Examples
+    --------
+    We can fit an input data using orthogonal distance regression (ODR) with
+    a polynomial model:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy import odr
+    >>> from scipy.odr.models import polynomial
```

Comment:
`polynomial` is also available in the `scipy.odr` namespace, so that should be preferred. can you just delete this line, and then further down use `odr.polynomial(3)`?

Reply:
That is better. I fixed it.